model/ConceptPropertyValue.php

```diff
@@ -33,12 +33,7 @@ public function __construct($model, $vocab, $resource, $prop, $clang = '')
 
     public function __toString()
     {
-        $label = is_string($this->getLabel()) ? $this->getLabel() : $this->getLabel()->getValue();
-        if ($this->vocab->getConfig()->sortByNotation()) {
-            $label = ltrim($this->getNotation() . ' ') . $label;
```

Comment:
Adding the notation here doesn't seem to have any effect on the sort order so I just removed it.